src/utils/media-utils.ts

```diff
@@ -57,7 +57,7 @@ export const getMediaFileInfo = async ({
       }
     } catch (err) {
       // If an error occurs while retrieving the image, we log but continue to return generic values
-      logger.error(err)
+      logger.error(JSON.stringify(err))
```

Comment:
Hmm what do we feel about wrapping a `JSON.stringify()` inside our logger codes, so the callers dont have to worry about converting JSON objects to string but can directly call the logger.error()?

Reply:
not inclined to do this - we define a message format (`Loggable`) that our log functions have to adhere to. This `JSON.stringify` is required because the error type here is `unknown`; if we were to allow `unknown` (and hence, shift this away from the client), it defeats the purpose of having a structured log 

i thought about exposing a `coerce = JSON.stringify` but given that it does nothing, i didn't add it in in the end

Reply:
![Screenshot 2023-05-31 at 1 42 19 PM](https://github.com/isomerpages/isomercms-backend/assets/42832651/7cc9164d-1f18-4799-85e6-6ae0dc7f4437)
Note that we do log the raw errors at multiple locations, so this would need to be changed at number call sites ya

Reply:
interesting - this means we have a missing step in our ci (and also tests for these)